docs/open-source-project/release-notes/tizen-4-0-m3.md

```diff
@@ -0,0 +1,258 @@
+  - Support for the Evas GL capability test has been added.
+  - EGL Sync is added for EvasGL’s EGLImage.
+
+### Fixes
+
+- DALi (3D UI Toolkit)
+  - The blending factor of premultiplied alpha format has been fixed.
+  - The depth index bug has been fixed in 3D layer.
+  - The bug where a depth buffer is sometimes not cleared has been fixed.
+  - The bug where a render task is waiting forever has been fixed.
+  - The system language change bug has been fixed.
+  - When same child object is added to same parent object, an Exception was occurred. This bug has been fixed. (C# NUI)
+  - Bug of memory leak problem caused by wrong increased ref-cnt has been fixed. (C# NUI)
+  - Bug of View's SiblingOrder has been fixed. (C# NUI)
+  - Bug of crash when getting Parent property as View has been fixed. (C# NUI)
+- Evas Render Engine Enhancement
+  - The resource sharing issue is fixed in Evas TBM backend.
+
+
+## UI Framework
+
+### New and changed features
+
+- EFL
+  - Assist-panel is added.
+  - Rotary Selector support edit mode.
+- Voice Framework
+  - Voice engine model is changed to a service app for supporting download (Only for mobile and TV profile).
+
+
+## Multimedia Framework
+
+### New and changed features
+
+- Camera
+  - New API set for hue setting have been added.
+  - Maximum supported number of camera is changed from 2 to 10.
+  - Pre-condition of setting ROI area API has been changed.
+- Media Content
+  - New Features are added.
+    - http://tizen.org/feature/content.scanning.others for scanning OTHERS-type files which are not included in the media content types such as IMAGE, VIDEO, SOUND or MUSIC.
+    - http://tizen.org/feature/content.filter.pinyin for supporting the Pinyin Filter. 
+- Media Controller
+  - In the mobile profile, repeat mode on for one media enumeration has been added.
+  - In the mobile profile, new API set for supporting playlist have been added.
+  - In the mobile profile, new API set for sending command and receiving command reply have been added.
+  - In the mobile profile, new API set for sending custom event and receiving event reply have been added.
+  - In the mobile profile, the API set for supporting metadata have been changed.
+  - In the mobile profile, the API set for sending custom command and receiving custom reply have been changed.
+  - In the mobile profile, the callbacks for receiving update information have been changed.
+  - In the mobile profile, some playback states enumerations have been changed.
+- Player
+  - Pre-condition of setting ROI area API has been changed.
+- Muse
+  - Support the on-demand launching using socket activation.
+  - Add initial gtest codes.
+  - Update message IPC mechanism.
+
+
+## Network and Connectivity
+
+### Fixes
+
+- Data Network
+  - A memory leak in the stc-manager has been fixed.
+  - The bug in stc-manager that caused a missing process in booting time has been fixed.
+  - CVE patches for Open-source packages have been applied.
```

Comment:
period is inconsistent in 194-196 and 198-202, either keep it or remove it

Reply:
198-202 : Period is needed as these are sentences.
194-196 : Do these need period? (I think it will be strange if period is used. Anyway please guide us.)